Neos.ContentRepository.Core/Classes/Feature/Common/NodeVariationInternals.php

```diff
@@ -256,6 +278,46 @@ protected function collectNodePeerVariantsThatWillHaveBeenCreated(
         return $events;
     }
 
+    private function resolveInterdimensionalSiblings(
```

Comment:
isnt this the same or can be combined with `resolveInterdimensionalSiblingsForCreation`?

Reply:
nope, creation operates on the explicitly set succeeding sibling, variation operates on the to-be-varied node itself